aeron-client/src/main/cpp/Aeron.h

```diff
@@ -413,6 +413,8 @@ class CLIENT_EXPORT Aeron
      */
     static std::string version();
 
+    static MemoryMappedFile::ptr_t mapCncFile(const std::string cncFileName, long mediaDriverTimeoutMs);
```

Comment:
This changes the API. Do not change the API without discussion.

Reply:
mapCncFile need to be accesed in Context::requestDriverTermination, maybe move it into Context is a good idea..

Reply:
The current version works just fine. There does not need to be a new API method for it.